wpilib/wpilib/sendablebase.py

```diff
@@ -15,6 +15,9 @@
 
 
 class SendableBase(Sendable):
+    """
+    """
```

Comment:
Could you please put a sensible docstring here? This is intended to be documented, so uh.

Also, you might like to send a PR upstream to fix their docs. Or I guess just use whatever upstream have right now. (Python teams being just as confused as Java teams isn't really our problem, right?)

Reply:
Oop, there was a docstring